server/services/prisonerSearchService.ts

```diff
@@ -1,14 +1,16 @@
-import type { Prisoner } from 'prisonRegisterApiClient'
+import type { PrisonerSummary } from 'viewModels'
 import { HmppsAuthClient, PrisonerSearchClient } from '../data'
+import toPrisonerSummary from '../data/mappers/prisonerSummaryMapper'
 
 export default class PrisonerSearchService {
   constructor(
     private readonly hmppsAuthClient: HmppsAuthClient,
     private readonly prisonerSearchClient: PrisonerSearchClient,
   ) {}
 
```

Comment:
This change is not directly related to `PrisonerListService`, but is to do with `PrisonerSearchService`
All of our other services deal with a REST API via it's client class which returns the type from the API definition, and then maps it into a view model or similar. The mapping is in the service method, and the service method does not expose the type from the API definition.

`PrisonerSearchService` did not do that - it exposed and returned the type from the REST API definition, and expected it's consumers to map it into something like a view model

This change (and the associated changes to its test class and its consuming class `routerRequestHandler`) correct that and change this service so that it adopts the patterns we use for all our other services